keyboards/planck/keymaps/sigul/keymap.c

```diff
@@ -0,0 +1,384 @@
+/*
+ * 
+ * An Italian ANSI layout
+ * Version 0.3 
+ * 
+ * Created by Silvio Gulizia on the basis of the default Planck keymap.
+ * thanks to SomeBuddyOnReddit, gepeirl, fauxpark, BXO511,  ...
+ *
+ * based on the original Planck layout
+ * Italian accented vowels "ò" and "à" have been moved from the QWERTY layer to the LOWER layers, while "è" and "ù" remain respectively on RAISE and LOWER.
+ *
+ */
+
+#include QMK_KEYBOARD_H
+#include "muse.h"
+#include <keymap_italian_osx_ansi.h>
+
+extern keymap_config_t keymap_config;
```

Comment:
```suggestion
```

Reply:
why this? I don't understand this code, it was included in the original planck and I kept it here.

Reply:
It's not needed as it should already be externed through one of the includes provided by `QMK_KEYBOARD_H`.